timely/src/progress/frontier.rs

```diff
@@ -317,6 +317,13 @@ impl<T: PartialOrder+Ord+Clone> MutableAntichain<T> {
         self.frontier().less_equal(time)
     }
 
+    /// Clones the vector of updates.
+    /// Only used for debugging purposes.
+    #[inline]
+    pub fn updates(&self) -> Vec<(T, i64)> {
```

Comment:
Could this return a `&[(T, i64)]` instead, to avoid a mandatory allocation?

Reply:
Removed the function completely as it's no longer needed.